src/codecs/music_gme.c

```diff
@@ -547,6 +547,7 @@ Mix_MusicInterface Mix_MusicInterface_GME =
     NULL,   /* LoopEnd [MIXER-X]*/
     NULL,   /* LoopLength [MIXER-X]*/
     GME_GetMetaTag,/* GetMetaTag [MIXER-X]*/
+    NULL,   /* GetUserTa [MIXER-X-snstruthers]*/
```

Comment:
A typo: `GetUserTa`. Did you meant the `GetUserTag`?

Reply:
LOL